pymaker/__init__.py

```diff
@@ -417,6 +418,12 @@ def _gas(self, gas_estimate: int, **kwargs) -> int:
         else:
             return gas_estimate + 100000
 
+    def _gas_replacement_bump(self) -> float:
+        if self._is_parity():
+            return 1.125
+        else:
+            return 1.1
+
```

Comment:
I'm not sure if this is ideal: even if your local node is geth and not parity, and will accept a replacement transaction with bump < 12.5%, then the parity nodes on the rest of the network will still refuse that replacement transaction. Seeing as the goal is presumably to have maximum network propagation, rather than to just get the local node to accept it, and a non-negligible number of nodes are parity, I think this should be `1.125` in all cases. Thoughts @EdNoepel ?

Reply:
That makes sense; much thanks for taking a look.